sansio_lsp_client/client.py

```diff
@@ -44,6 +66,87 @@ class ClientState(enum.Enum):
     SHUTDOWN = enum.auto()
     EXITED = enum.auto()
 
+#NOTE: Server capabilitie: resolveProvider - info about completion item -- can't use?
+CAPABILITIES = {
+    'textDocument': {
+        'synchronization': {
+            'didSave': True,
+            #'willSaveWaitUntil': True,
+            'dynamicRegistration': True,
+            #'willSave': True
+        },
+
+        'completion': {
+            'dynamicRegistration': True,
+            'completionItem': {
+                'snippetSupport': False
+            },
+            'completionItemKind': {
+                'valueSet': [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25]
+            }
+        },
+        'hover': {
+            'dynamicRegistration': True,
+            'contentFormat': ['markdown', 'plaintext'],
+        },
+        'definition': {
+            'dynamicRegistration': True,
+            'linkSupport': True,
+        },
+        'signatureHelp': {
+            'dynamicRegistration': True,
+            'signatureInformation': {
+                'parameterInformation': {
+                    'labelOffsetSupport': False
+                },
+                'documentationFormat': ['markdown', 'plaintext']
+            }
+        },
+        'implementation': {
+            'linkSupport': True,
+            'dynamicRegistration': True,
+        },
+        'references': {
+            'dynamicRegistration': True
+        },
+        'callHierarchy':{
+            'dynamicRegistration': True
+        },
+        'declaration': {
+            'linkSupport': True,
+            'dynamicRegistration': True
+        },
+        'typeDefinition': {
+            'linkSupport': True,
+            'dynamicRegistration': True
+        },
+
+        'formatting': {
+            'dynamicRegistration': True
+        },
+        'rangeFormatting': {
+            'dynamicRegistration': True
+        },
+
+        'documentSymbol': {  # Document Symbols Request
+            'hierarchicalDocumentSymbolSupport': True,
+            'dynamicRegistration': True,
+            'symbolKind': {
+                'valueSet': [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26]
```

Comment:
`list(some_int_enum)` gives a list of all its members (as enums, but they should be compatible with `int`):

```suggestion
                'valueSet': list(SymbolKind)
```